functions/Get-DbaBackupInformation.ps1

```diff
@@ -144,6 +147,8 @@ function Get-DbaBackupInformation {
         [parameter(ParameterSetName = "Create")]
         [Switch]$NoXpDirTree,
         [parameter(ParameterSetName = "Create")]
+        [Switch]$NoXpDirRecurse = $false,
```

Comment:
Isn't $false always the default for a swithch?

Reply:
yes, this does seem redundant